invest/templates/invest/high-potential-opportunities-form.html

```diff
@@ -0,0 +1,25 @@
+{% extends 'core/base.html' %}
+{% load static from staticfiles %}
+
+{% load add_export_elements_classes from directory_components_tags %}
+ {% block head_css %}
+    {{ block.super }}
+    <link href="{% static 'styles/high-potential-opportunities-form.css' %}" rel="stylesheet" type="text/css" />
+{% endblock %}
+ {% block head_title %}
+    High Potential Opportunities - great.gov.uk
```

Comment:
should this not be in CMS?

Reply:
yeah, in fact the CMS already supports that field